### ISSUE TITLE
Schema error paths should include Resource type as context

### ISSUE THREAD

**Issue by jmandel** [microsoft/health-cards-validation-SDK#55]
For example in this error output:

```
             └─ Error : 
                  Schema: {"keyword":"type","dataPath":"/name/0/family","schemaPath":"#/definitions/string/type","params":{"type":"string"},"message":"should be string"}
                  Schema: {"keyword":"type","dataPath":"/category","schemaPath":"#/properties/category/type","params":{"type":"array"},"message":"should be array"}

```

It's hard to tell that the first schema error is about **Patient**.name.family, and the second is about **Observation**.category

**Comment by ljoy913** [comment]
I updated this yesterday in the pending 0.4.4 update:
```
    └─ Error :
              Schema: {"keyword":"type","dataPath":"/entry/0/name/0/family","schemaPath":"#/definitions/string/type","params":{"type":"string"},"message":"should be string"}
```         
I used __entry/index__ instead of __Resource type__ as there may be multiple Resources of the same type.  I could mix in the Resource type as well like "dataPath":"/entry/0/Patient/name/0/family" - any preference?

**Comment by jmandel** [comment]
I think including the resource type somewhere as well as the entry index
would be helpful. This syntax that you suggested here looks good to me.

**Comment by christianpaquin** [comment]
Should be fixed by #49; @jmandel I'll let you check and close the issue.
